wom/src/main/scala/wom/values/WomValue.scala

```diff
@@ -68,6 +69,8 @@ trait WomValue {
       case w => symbolHash(w.valueString)
     }
   }
+
+  def asWomExpression: WomExpression = new ValueAsAnExpression(this)
```

Comment:
perhaps `ValueAsWomExpression`

Reply:
does this def add value over calling `new ValueAsAnExpression` directly?

Reply:
@cjllanwarne It's for convenience, so IMO yes.